### PR TITLE
feat(Conversation): mark file/object shares with icon in conversations list

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Show icons next to last message

### 🖼️ Screenshots / Screencasts

☀️ Light theme | 🌑 Dark Theme
-- | --
image | image
image | 

### 🏁 Checklist

- [ ] 🌏 Tested with different browsers / clients:
  - [x] Chromium (Chrome / Edge / Opera / Brave)
  - [ ] Firefox
  - [ ] Safari
  - [ ] Talk Desktop
  - [ ] Not risky to browser differences / client
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/composables/useConversationInfo.js [hunk 1]
```diff
@@ -55,7 +56,7 @@ export function useConversationInfo({
 		}
 
 		const params = item.value?.lastMessage.messageParameters
-		let subtitle = item.value?.lastMessage.message.trim()
+		let subtitle = (getMessageIcon(item.value?.lastMessage) + ' ' + item.value?.lastMessage.message).trim()
```
Comment (Antreesy) [inline src/composables/useConversationInfo.js:59]: There is one usage of unescaped result:

```js
		if (shortLastChatMessageAuthor.value === '') {
			return simpleLastChatMessage.value
		}
```

File: src/components/LeftSidebar/ConversationsList/Conversation.vue [hunk 2]
```diff
@@ -21,7 +21,8 @@
 			<ConversationIcon :item="item" :hide-favorite="false" :hide-call="false" />
 		</template>
 		<template #subname>
-			{{ conversationInformation }}
+			<!-- eslint-disable-next-line vue/no-v-html -->
+			<span v-html="conversationInformation" />
```
Comment (DorraJaouad) [inline src/components/LeftSidebar/ConversationsList/Conversation.vue:25]: No? 
The message | the LS 
-- | --
image | image

File: src/composables/useConversationInfo.js (hunk 1 above)
Comment (DorraJaouad) [inline src/composables/useConversationInfo.js:59]: They should be split

**Review by DorraJaouad — APPROVED** [review]
Tested ✅

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**